processing/src/main/java/org/apache/druid/error/DruidException.java

```diff
@@ -129,7 +131,7 @@
  * introduced DruidException.
  */
 @NotThreadSafe
-public class DruidException extends RuntimeException
+public class DruidException extends RuntimeException implements SanitizableException
```

Comment:
lets revert the changes here. I don't think the whole `SanitizableException` was well thought out and don't want DruidException to extend it. In an ideal world, we shouldn't have SanitizableException and just use DruidException

Reply:
I fully agree, but `SanitizableException` and `ErrorResponseTransformStrategy` are coupled. To work around this without having DruidException , I've had to create a new function in `ErrorResponseTransformStrategy`. Let me know if this is fine.